src/plugins/kibana_react/public/page_template/page_template.tsx

```diff
@@ -62,8 +87,14 @@ export const KibanaPageTemplate: FunctionComponent<KibanaPageTemplateProps> = ({
   return (
     <EuiPageTemplate
       template={template}
-      pageHeader={pageHeader}
       restrictWidth={restrictWidth}
+      paddingSize={template === 'centeredBody' ? 'none' : 'l'}
+      pageHeader={pageHeader}
+      pageSideBar={pageSideBar}
+      pageSideBarProps={{
+        className: classNames('kbnPageTemplate__pageSideBar', rest.pageSideBarProps?.className),
+        ...rest.pageSideBarProps,
```

Comment:
Do we need to actually destructure `pageSideBarProps.className` here to remove `className` from `pageSideBarProps`?

I'm thinking if someone passes a `className` then won't it override your custom one because it's still in the props.

Reply:
Ah yeah, I just need to reverse these and have className come last. Good catch!

```suggestion
        ...rest.pageSideBarProps,
        className: classNames('kbnPageTemplate__pageSideBar', rest.pageSideBarProps?.className),
```